src/Console/Command/FixCommand.php

```diff
@@ -157,6 +157,9 @@ public function getHelp(): string
 
 * <comment>none</comment>: disables progress output;
 * <comment>dots</comment>: multiline progress output with number of files and percentage on each line.
+* <comment>detail</comment>: multiline progress output with status, path and applied fixers for each file.
+* <comment>meaningful</comment>: multiline progress output with status, path and applied fixers for each file with meaningful result (non-modified or skipped are ignored).
+* <comment>Custom\FQCN</comment>: FQCN to a class implementing PhpCsFixer\Console\Output\Progress\ContextAwareOutputFactoryInterface that will handle the output.
```

Comment:
please split. for new formatter, it's easy to review and merge. for custom formatted, we will have proper discussion about solution we may want to introduce.

Reply:
Support for custom fixers was removed from this PR and will be provided in separate one.

Reply:
custom fixers? 🤔

Reply:
Custom printers of course 😉.